packages/core/src/temporal-types.ts

```diff
@@ -359,8 +359,24 @@ export class Date<T extends NumberOrInteger = Integer> {
    *
    * @param {global.Date} standardDate - The standard JavaScript date to convert.
    * @return {Date} New Date.
+   * @deprecated use {@link fromStandardDateLocal} which is a drop in replacement, or {@link fromStandardDateUTC} which takes the Year, Month and Date from UTC rather than Local time
    */
   static fromStandardDate (standardDate: StandardDate): Date<number> {
+    return this.fromStandardDateLocal(standardDate)
+  }
+
+  /**
+   * Create a {@link Date} object from the given standard JavaScript `Date` using the Year, Month and Date in Local Time.
+   * Hour, minute, second, millisecond and time zone offset components of the given date are ignored.
+   *
+   * NOTE: this function and {@link toStandardDate} are not inverses of one another.
+   * This takes the Day, Month and Year in local time from the supplies JavaScript Date object, while {@link toStandardDate} creates a new JavaScript Date object at midnight UTC.
```

Comment:
```suggestion
   * This takes the Day, Month and Year in local time from the supplied JavaScript Date object, while {@link toStandardDate} creates a new JavaScript Date object at midnight UTC.
```
I see this is a copy-pasta, so consider fixing the existing doc comment as well.